example/package.json

```diff
@@ -3,12 +3,18 @@
   "version": "1.0.0",
   "description": "Showing how serverless-plugin-ignitor can be used",
   "scripts": {
-    "deploy": "sls deploy",
-    "local:ignitor": "sls invoke local -f hello -p fixtures/ignitor.json",
-    "local": "sls invoke local -f hello -p fixtures/event.json"
+    "deploy": "serverless deploy",
+    "local:hello": "serverless invoke local --function hello --p fixtures/ignitor.json",
```

Comment:
Adding another function to show the difference between a wrapped function and non-wrapped function.